arrus/core/api/devices/us4r/Us4R.h

```diff
@@ -179,6 +179,34 @@ class Us4R : public DeviceWithComponents {
     */
     virtual void setActiveTermination(std::optional<uint16> value) = 0;
 
+    /**
+    * Enables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void enableAfeAutoOffsetRemoval(void) = 0;
+
+    /**
+    * Disables AFE auto offset removal.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void disableAfeAutoOffsetRemoval(void) = 0;
+
+    /**
+    * Sets AFE auto offset removal accumulator cycles number.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void setAfeAutoOffsetRemovalCycles(uint16_t cycles) = 0;
+
+    /**
+    * Sets AFE auto offset removal delay from TX_TRIG.
+    *
+    * See docs of arrus::devices:: for more information.
+    */
+    virtual void setAfeAutoOffsetRemovalDelay(uint16_t delay) = 0;
```

Comment:
What is the default removal delay (i.e. value that will be used, when this method is not called)?

Reply:
Default = 0, added to comment